applications/launchpad_v2/src/containers/WalletContainer/TariWallet/Smiley.tsx

```diff
@@ -0,0 +1,99 @@
+import { SVGProps } from 'react'
+
+const Smiley = ({
+  on,
+  ...props
+}: { on: boolean } & SVGProps<SVGSVGElement>) => {
+  if (on) {
+    return (
+      <svg
+        width='20'
+        height='20'
```

Comment:
When toggling the wallet id, these icons goes slightly up and down (I guess by 1px). Probably it's because of the different sizes of these svgs (`20` and `21`). You can see this on my video. Not sure if overriding width and height here will help. Maybe we need a better export from Figma, so both icons have the same size and viewbox.